java/daikon/dcomp/DCRuntime.java

```diff
@@ -98,6 +107,17 @@
   /** If true, merge arrays and their indices. */
   private static boolean merge_arrays_and_indices = true;
 
+  /** Decl writer to share output code with Chicory. */
+  // Set in Premain.premain().
+  static DeclWriter decl_writer;
+
+  /** Instance of DCRuntime to use for IComparability callbacks. */
+  // Set in Premain.premain().
+  static DCRuntime runtime_object;
+
+  /** Flag that indicates when the first class has been processed. */
```

Comment:
Actually this seems to indicate when the first class has *not* been processed.
Can you improve its name and documentation?

Also, please don't introduce *new* identifiers in snake_case.